scipy/stats/_stats_py.py

```diff
@@ -331,6 +335,11 @@ def hmean(a, axis=0, dtype=None):
     Use masked arrays to ignore any non-finite values in the input or that
     arise in the calculations such as Not a Number and infinity.
 
+    References
```

Comment:
Good addition although a foundational article would be better (or both). Optional

Reply:
Added.